plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -127,8 +133,9 @@ typedef struct wrkrInstanceData {
 /* tables for interfacing with the v6 config system */
 /* action (instance) parameters */
 static struct cnfparamdescr actpdescr[] = {
-	{ "server", eCmdHdlrGetWord, 0 },
+	{ "server", eCmdHdlrArray, 0 },
```

Comment:
I did not add a test where server is specified as just a string, not an array, since presumably this is handled by the grammar itself.